docker/utils/utils.py

```diff
@@ -100,7 +101,14 @@ def tar(path, exclude=None, dockerfile=None, fileobj=None):
     exclude = exclude or []
 
     for path in sorted(exclude_paths(root, exclude, dockerfile=dockerfile)):
-        t.add(os.path.join(root, path), arcname=path, recursive=False)
+        i = t.gettarinfo(os.path.join(root, path), arcname=path)
+        if sys.platform == 'win32':
+            i.mode = i.mode & 0o755 | 0o111
```

Comment:
This looked weird, but it does match `pkg/archive/archive_windows.go` in docker/docker, so I guess we should keep it this way. Not sure why write is stripped from group/other.